### PR TITLE
Support symfony 7

### PR DESCRIPTION
**Pull request by Wiktor6**
AC
---
* support symfony 7
* drop support symfony < 6

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,24 +9,24 @@
...
-        "phpunit/phpunit": "^9.6.5",
+        "friendsofphp/php-cs-fixer": "^v3.64",
+        "matthiasnoback/symfony-config-test": "^5.2",
+        "phpro/grumphp": "^1.16",
+        "phpstan/phpstan": "^1.12",
+        "phpstan/phpstan-webmozart-assert": "^1.2",
+        "phpunit/phpunit": "^10.5",
         "roave/security-advisories": "dev-latest",
-        "symfony/phpunit-bridge": "^6.2.7"
+        "symfony/phpunit-bridge": "^6.4|^7.0"
```
Comment (lukasz-falda) [inline composer.json:29]: why 6|7? This is for develop, use newest possible

File: phpunit.xml.dist [hunk 2]
```diff
@@ -1,23 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.5/phpunit.xsd"
```
Comment (lukasz-falda) [inline phpunit.xml.dist:3]: use file from vendor

File: phpunit.xml.dist [hunk 3]
```diff
@@ -1,23 +1,18 @@
...
          backupGlobals="false"
          colors="true"
-         convertDeprecationsToExceptions="false"
+         cacheDirectory=".phpunit.cache"
```
Comment (lukasz-falda) [inline phpunit.xml.dist:6]: is this not default or something?

**Review by lukasz-falda — APPROVED** [review]

**Review by bmulawa — APPROVED** [review]

File: phpunit.xml.dist (hunk 3 above)
Comment (Wiktor6) [reply phpunit.xml.dist:6]: According to the documentation there is no default param for this one https://docs.phpunit.de/en/11.3/configuration.html#the-cachedirectory-attribute.

**Pull request merged**